tvm-tac/src/main/kotlin/org/ton/tac/TacGeneration.kt

```diff
@@ -159,63 +211,26 @@ private fun TvmControlFlowContinuation.hasStandardC0Save(): Boolean {
 }
 
 // special cases
-private fun TvmInst.ignoreBranches(): Boolean = this is TvmDictSpecialDictigetjmpzInst
-
-private fun <Inst : AbstractTacInst> processOrdinaryInst(
-    ctx: TacGenerationContext<Inst>,
-    stack: Stack,
-    inst: TvmRealInst,
-    endingInstGenerator: EndingInstGenerator<Inst>,
-): List<Inst> {
-    val operands = extractPrimitiveOperands(inst)
-
-    val specInputs = inst.stackInputs ?: emptyList()
-    val specOutputs = inst.stackOutputs ?: emptyList()
-
-    val operandContinuationInfo =
-        if (inst is TvmContOperandInst) {
-            extractOperandContinuations(ctx, inst)
-        } else {
-            null
-        }
-
-    val (inputs, outputs, stackContinuationMap) =
-        stack.processNonStackInst(
-            ctx,
-            inputSpec = specInputs,
-            outputSpec = specOutputs,
-            contRef = operandContinuationInfo?.resultContinuationId,
-        )
-
-    if (inst.branches.isEmpty() || inst.ignoreBranches()) {
-        val tacInst =
-            TacOrdinaryInst<Inst>(
-                mnemonic = inst.mnemonic,
-                operands = operands,
-                inputs = inputs.map { it.second },
-                outputs = outputs,
-                blocks = emptyList(),
-            )
-
-        val result =
-            if (ctx.debug) {
-                val stackAfter = stack.copyEntries()
-                TacInstDebugWrapper(tacInst, stackAfter)
-            } else {
-                tacInst
-            }
-
-        @Suppress("unchecked_cast")
-        return listOf(result as Inst)
-    }
-
-    val continuationMap =
-        operandContinuationInfo?.let {
-            it.operandContinuationIds + stackContinuationMap
-        } ?: stackContinuationMap
-
-    throwErrorIfBranchesNotTypeVar(inst)
-
+private fun TvmInst.ignoreBranches(): Boolean =
+    this is TvmDictSpecialDictigetjmpzInst ||
+        this.mnemonic == "CALLDICT" ||
+        this.mnemonic == "CALLDICT_LONG"
```

Comment:
Here, it is also better not to use mnemonics, but to use `is Tvm...` expressions